src/Core/Geographic/Coordinates.js

```diff
@@ -137,7 +137,10 @@ function _convert(coordsIn, newCrs, target) {
     } else {
         if (coordsIn.crs === 'EPSG:4326' && newCrs === 'EPSG:4978') {
             const cartesian = ellipsoid.cartographicToCartesian(coordsIn);
-            return target.set(newCrs, cartesian);
+            target.set(newCrs, cartesian.position);
+            // store the normal vector of the point.
+            target._surfaceNormal = cartesian.normal;
```

Comment:
Again.
Please reverse the logic here so we don't need to ever modify `target._surfaceNormal` outside of the property def.
```js
const cartesian = ellipsoid.cartographicToCartesian(coordsIn, coordsIn.geodeticSurfaceNormal);
target.set(newCrs, cartesian.position);
```